src/Config/ConfigFileLoader.php

```diff
@@ -0,0 +1,56 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Psh\Config;
+
+abstract class ConfigFileLoader implements ConfigLoader
+{
+
+    /**
+     * @param string $file
+     * @return string
+     */
+    protected function loadFileContents(string $file): string
+    {
+        return file_get_contents($file);
+    }
+
+    /**
+     * @param string $applicationRootDirectory
+     * @param string $absoluteOrRelativePath
+     * @param string $baseFile
+     * @return string
+     */
```

Comment:
der php doc block kann von mir aus weg. Steht ja alles schon in der Methoden Signatur drin :-)